node/src/accountant/mod.rs

```diff
@@ -2431,14 +2472,29 @@ mod tests {
         System::current().stop();
         assert_eq!(system.run(), 0);
         let mut notify_later_params = notify_later_params_arc.lock().unwrap();
-        let (msg, interval) = notify_later_params.remove(0);
+        // As obvious, the next scan is scheduled for the future and should not run immediately.
+        let (msg, actual_interval) = notify_later_params.remove(0);
         assert_eq!(
             msg,
             ScanForNewPayables {
                 response_skeleton_opt: None
             }
         );
-        assert_eq!(interval, Duration::from_secs(500));
+        let default_scan_intervals = ScanIntervals::compute_default(TEST_DEFAULT_CHAIN);
+        // The previous last_new_payable_scan_timestamp is UNIX_EPOCH, if the interval was derived
+        // from that timestamp, it would result in an immediate-scan command. This implies that
+        // the last_new_payable_scan_timestamp was reset to zero, which is how it is meant to be.
```

Comment:
Please rewrite this comment in the way:

```
        // Since no immediate scan was triggered, the timestamp was properly reset to zero
        // (rather than using UNIX_EPOCH which would cause immediate scheduling).
```